build.gradle

```diff
@@ -15,6 +16,11 @@ allprojects {
         google()
         jcenter()
     }
+
+    tasks.withType(Javadoc) {
```

Comment:
To prevent javadoc build error: https://stackoverflow.com/questions/34828426/disable-javadoc-check-for-bintray-upload